src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Simplify kubernetes layers with New Relic
+---
+
+import k8sDiagramContext from 'images/tutorials_diagram_kubernetes-overview.webp'
+
+import tutorialOverviewdashboard from 'images/tutorial_screenshot-full_overview-dashboard.webp'
+
+import tutorialOverviewDeployments from 'images/tutorial_screenshot-full_overview-deployments.webp'
+
+import tutorialOverviewFailed from 'images/tutorial_screenshot-crop_failed-pods.webp'
+
+import tutorialAPMOverview from 'images/tutorial_screenshot-full_apmK8s.webp'
+
+import tutorialAPMPerformance from 'images/tutorial_screenshot-full_apm-k8s-performance.webp'
+
+Understanding the inner workings of your Kubernetes system, specifically how all the different functions of the system come together, is inherently complex. Let's review what makes up a Kubernetes system and then dive into how New Relic can help monitor and understand each part.
+
+## Break it apart to understand
+
+To gain a better grasp of a Kubernetes system, let's break it down into distinct layers and explore how New Relic can help you comprehend each one effectively.
+
+<img
+    title="k8sDiagramContext"
+    alt="an image showing an abstracted view of a kubernetes system. This includes pods, apps, and clusters"
+    src={k8sDiagramContext}
+/>
+
+We'll discuss a Kubernetes system in three key sections:
+
+  * **The Cluster**: This represents the entire Kubernetes system. Imagine a vast bakery comprised of many ovens each with its own set of bakers. The cluster contain multiple deployments, which in turn house many pods, with each pod containing its individual services and applications.
+
+  * **The Orchestrated**: These are the core elements of a Kubernetes system. Think of them as the individual baking stations within the bakery. Orchestrated components consist of deployments that, in turn, house pods, working together in harmony.
+
+  * **The Services and Applications**: Services and applications are the workhorses of the Kubernetes system. Think of them as the indivdual bakers running each oven. Within a Kubernetes system each pod houses one or more services and applications. The services and applocations provide the essential functionality that drives the system. This could be computation, a web application, or any other application.
+
+It's important to note that these sections interalate with each other. The cluster contains multiple orchestrated layers, and each orchestrated section consists of multiple service and application layers.
+
+##  Understand and monitor the cluster layer
+
+Imagine your Kubernetes cluster as an expansive bakery again. The bakery is equipped with multiple ovens and skilled bakers staioned on each oven. If your bakery has only two or three ovens and a few bakers, it's manageable to oversee each oven unit personally. However, when we scale up with dozens or hundreds of bakers and ovens, keeping track of every single unit becomes a near-impossible task.
```

Comment:
"expansive" is an odd word choice

Reply:
I'm not sure this bakery metaphor is really helping you out.... (not sure why it added Pending there)